lib/schedule.js

```diff
@@ -2,6 +2,8 @@
 
 var each = require('@ndhoule/each');
 
+var CLOCK_LATE_FACTOR = 2;
```

Comment:
I think this factor could go higher if we want to, currently checkReclaim runs every 3 seconds but we only action queues that havent been acknowledged for 10 seconds